src/Compilers/Core/Portable/Collections/BitVector.cs

```diff
@@ -205,7 +207,17 @@ public static BitVector AllSet(int capacity)
         /// <returns></returns>
         public BitVector Clone()
         {
-            return new BitVector(_bits0, (_bits == null) ? null : (_bits.Length == 0) ? s_emptyArray : (Word[])_bits.Clone(), _capacity);
+            Word[] newBits;
+            if (_bits is null || _bits.Length == 0)
+            {
+                newBits = s_emptyArray;
```

Comment:
📝 This is a behavior change if `_bits` is null. I'm assuming that's intentional?

Reply:
Yes. The intent of the type is to use `s_emptyArray` in the case of a zero length `BitVector`. The `null` case is just there to handle `default(BitVector)`. This should be unobservable to callers.

---
In reply to: [356224544](https://github.com/dotnet/roslyn/pull/40268#discussion_r356224544) [](ancestors = 356224544)